docs/source/daml/intro/2_Scenario.rst

```diff
@@ -63,6 +64,7 @@ In DAML Studio, you should see the text "Scenario results" just above the line `
 
 This opens the scenario view in a separate column in VS Code. The default view is a tabular representation of the final state of the ledger:
 
+.. Unsure how to add new image correctly, but the below image doesn’t match up with what should be read. It shouldn’t read Intro_2_Scenario. It should read Token_Test, as described after the image. Please change accordingly with all images in this section.
```

Comment:
Just take a correct screenshot and replace the png file. I can help you with that if you like.